### ISSUE TITLE
Gamma correction

### ISSUE THREAD

**Issue by toothbrush7777777** [phip1611/noto-sans-mono-bitmap-rs#24]
Should the font image values be gamma-corrected for standard displays (i.e. sRGB) instead of being linear intensities? In other words, should the values be `linear_to_srgb(intensity)`?

It isn't very hard to filter the values when creating the font images, and would be faster than doing so upon reading them.

The characters look better and less blurry after gamma correction for sRGB displays.

**Comment by phip1611** [comment]
Sounds correct, yes. TBH, I never thought about this.

**Comment by toothbrush7777777** [comment]
Yeah, I messed around with this a bit when I was writing a renderer a few months ago.
I noticed it particularly on low-PPI displays.

**Comment by phip1611** [comment]
> Shall I create a pull request for that?

Yes, I will appreciate that very much!
